transport/tchannel/handler_test.go

```diff
@@ -805,6 +805,32 @@ func TestTruncatedHeader(t *testing.T) {
 	}
 }
 
+func TestRpcServiceHeader(t *testing.T) {
+	hw := &handlerWriter{}
+	h := handler{
+		headerCase: canonicalizedHeaderCase,
+		newResponseWriter: func(_ inboundCallResponse, _ tchannel.Format, _ headerCase) responseWriter {
```

Comment:
nit: if all are ignored with _, you don't need to specify:
`func( inboundCallResponse,  tchannel.Format,  headerCase) responseWriter`

Reply:
updated. thanks